modules/acs-cloud-requirements.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * cloud_service/acscs-default-requirements.adoc
+:_mod-docs-content-type: CONCEPT
+[id="acs-cloud-requirements_{context}"]
+= General requirements for RHACS Cloud Service
+
+[role="_abstract"]
+{product-title-managed} has some system requirements that must be met before you can install it.
+
+[WARNING]
+====
+You must not install {product-title-managed-short} on:
+
+* Amazon Elastic File System (Amazon EFS). Use the Amazon Elastic Block Store (Amazon EBS) with the default *gp2* volume type instead.
+* Older CPUs that do not have the Streaming SIMD Extensions (SSE) 4.2 instruction set.
+For example, Intel processors older than _Sandy Bridge_ and AMD processors older than _Bulldozer_.
+These processors were released in 2011.
+====
+
+To install {product-title-managed-short}, you must have one of the following systems:
+
+* {ocp} version {ocp-supported-version} or later, and cluster nodes with a supported operating system of {op-system-first} or {op-system-base-full}
+* A supported managed Kubernetes platform, and cluster nodes with a supported operating system of Amazon Linux, CentOS, Container-Optimized OS from Google, {op-system-first}, Debian, {op-system-base-full}, or Ubuntu
++
+For information about supported platforms and architecture, see the link:https://access.redhat.com/articles/7045053[{product-title} Support Matrix].
+
+The following minimum requirements apply to cluster nodes:
+
+* *Architecture*: Supported architectures are `amd64`, `ppc64le`, or `s390x`.
++
+[NOTE]
+====
+Secured cluster services are supported on {ibm-power} (`ppc64le`), {ibm-z} (`s390x`), and {ibm-linuxone} (`s390x`) clusters.
+====
+* *Processor*: 3 CPU cores are required.
+* *Memory*: 6 GiB of RAM is required.
++
+[NOTE]
+====
+See the default memory and CPU requirements for each component and ensure that the node size can support them.
+====
+* *Persistent volume claim (PVC)*: A PVC is strongly recommended on the secured clusters where Scanner V4 is enabled. Use Solid-State Drives (SSDs) for best performance. However, you can use another storage type if you do not have SSDs available.
```

Comment:
```suggestion
* *Storage*: A persistent volume claim (PVC) is strongly recommended on secured clusters where Scanner V4 is enabled. Use solid-state drives (SSDs) for best performance. However, you can use another storage type if you do not have SSDs available.
```

Optional suggestion to consider: I wonder if definition lists or a table would be clearer than the current formatting.